### DISCUSSION TITLE
Will the gpu-utils support the new RDNA3 cards being released next month?

### DISCUSSION THREAD

**Discussion by KeithMyers** [Ricks-Lab/gpu-utils#138] [Q&A]
Will the gpu-utils support the new RDNA3 cards being released next month?

**Comment by Ricks-Lab** [comment]
I bought previous generation NAVI23 6xxx series and it worked with no modifications to the utilities.   Hopefully next generation is supported in Linux drivers on day one.  Only way to know for sure is to try it out.  I am available to troubleshoot if problems arise.

**Comment by Ricks-Lab** [comment]
@KeithMyers I have been digging around with latest generation AMD GPU and found there is a slight change to implementation of CurvePoints, with a voltage offset being specified instead of voltage end points.  I am working on the implementation now.

**Comment by Ricks-Lab** [comment]
@KeithMyers  @csecht 
I have a functional version that implements Vddgfx offset mode.  I had to make significant changes and touched code that could affect all modes.  Let me know if you see any issues.  Also, is the voltage offset useful with the Navi GPUs?  My only Navi GPU is remote in my lab in Portland, so I can only do limited testing so far.

The updated code is in a new branch: [pp_feature_refine](https://github.com/Ricks-Lab/gpu-utils/tree/pp_feature_refine)
